ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/session-timeout/session-timeout.tsx

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+
+import * as React from 'react'
+import styled from '../../styles/styled-components'
+const sessionTimeoutModel = require('../../../component/singletons/session-timeout')
+import { Button, buttonTypeEnum } from '../button'
+
+const SessionTimeoutRoot = styled.div`
+  height: 100%;
+  width: 100%;
+  display: block;
+  overflow: hidden;
+`
+const Message = styled.div`
+  max-height: 'calc(100% - 2.25rem)';
```

Comment:
I had to hardcode the value of `minimumButtonSize`(2.75rem) - 0.5rem. If i did the proper thing and pass in the props, (e.g. `${props=>props.theme...}`) the formatter would cause the line to be split between 2 lines, which would break the compiler. Likewise if I fixed it the formatter would cause the build to fail.

Reply:
Why is `calc(...)` in in single quotes?

Reply:
@djblue Is there a way to disable the formatter for this case?